release notes/v0.43.0.md

```diff
@@ -0,0 +1,125 @@
+k6 v0.43.0 is here! :tada:
+
+Notable changes in this release include:
+- [xk6-browser](https://github.com/grafana/xk6-browser) is now bundled in k6 as an experimental module, and usable without a separate binary or compilation step!
+- Native support for JavaScript's `async`/`await`.
+- A new experimental JavaScript module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+
+## New Features
+
+### xk6-browser is now a built-in module [#2884](https://github.com/grafana/k6/pull/2884)
+
+[xk6-browser](https://github.com/grafana/xk6-browser), a k6 extension that enables browser automation and end-to-end testing from k6 scripts, now comes bundled with k6 under the `k6/experimental/browser` import path. This makes it more accessible, as users no longer need to build their own binary with [xk6](https://github.com/grafana/xk6), or use a separate binary to run browser tests.
+
+All current xk6-browser scripts that work with v0.8.0 will continue to work with the built-in module in k6 v0.43.0, by simply changing the import path from `k6/x/browser` to `k6/experimental/browser`, and setting the environment variable `K6_BROWSER_ENABLED` to `true`. The environment variable is needed to minimize potential risks with launching browser or any other processes via k6 scripts. It's also a mechanism we use in the k6 Cloud, where browser tests are currently disabled.
+
+You can see a script example [here](https://github.com/grafana/k6/blob/v0.43.0/samples/experimental/browser.js), and read the updated [documentation](https://k6.io/docs/javascript-api/k6-browser/).
```

Comment:
```suggestion
You can see a script example [here](https://github.com/grafana/k6/blob/v0.43.0/samples/experimental/browser.js), and read the updated [documentation](https://k6.io/docs/javascript-api/xk6-browser/).
```
I don't see an open PR for it, if the browser team is planning it, you can skip the suggestion.

Reply:
Sorry, here we are https://github.com/grafana/k6-docs/pull/996